WebDriverAgentLib/Commands/FBAlertViewCommands.m

```diff
@@ -44,7 +45,13 @@ + (NSArray *)routes
 + (id<FBResponsePayload>)handleAlertAcceptCommand:(FBRouteRequest *)request
 {
   FBSession *session = request.session;
-  if (![[FBAlert alertWithApplication:session.application] acceptWithError:nil]) {
+  NSString *name = request.parameters[@"name"];
+
+  if ([name length] != 0) {
+    if (![[FBAlert alertWithApplication:session.application] clickAlertButton:name error:nil]) {
+      return FBResponseWithStatus(FBCommandStatusNoAlertPresent, nil);
```

Comment:
That might be misleading right? We should handle error here properly and message whether alert or this particular button is not present